src/commands/repos/UpdateRepos.ts

```diff
@@ -17,9 +18,22 @@ export class UpdateRepos extends AbstractReposCommand {
     protected resetToRemote: boolean;
 
     public async execute(): Promise<void> {
-        await Promise.all(
-            Object.keys(this.parentRepos).map((repoName) => this.handleRepo(repoName))
-        );
+        Global.isVerbose() && console.log('update repos - prepare');
+        await promiseAllSettled(
+            {
+                keys: Object.keys(this.parentRepos),
```

Comment:
Style: Where should be the line-breaks and what should the indentation be?
When the object braces are on the same line as the parenthesis (`await promiseAllSettled({`), the whole object body is automatically indented quite deep, and I don't really like that.

Reply:
I am ok with the indentation and also happy if we can use autoformat in Intellij